learning/katas/go/Core Transforms/Partition/Partition/go.mod

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module partition
+
+go 1.14
+
+require (
+	github.com/apache/beam v2.22.0+incompatible
```

Comment:
I realize that we're using multiple versions of Beam, from v2.19 to v2.22.
Can we standardize to just one version for the entire course?

Reply:
Thank you, @henryken for picking up on that.  I will add this to my checklist to remind me for every PR.